pkg/query-service/constants/constants.go

```diff
@@ -87,3 +98,44 @@ func GetOrDefaultEnv(key string, fallback string) string {
 	}
 	return v
 }
+
+var StaticInterestingLogFields = []model.LogField{
+	{
+		Name:     "trace_id",
+		DataType: "String",
+		Type:     Static,
+	},
+	{
+		Name:     "span_id",
+		DataType: "String",
```

Comment:
You can create a constant or enum for these set of datatypes.

Reply:
have converted them to constants